importers/data_sources/importer.py

```diff
@@ -0,0 +1,272 @@
+"""
+Loads canonical (defined in the codebase) or separate (defined elsewhere) 
+documents into the data_sources collection in the Relation Engine (RE).
+
+This is a simple importer, as the data_sources load data is quite simple,
+and also very similar to its ultimate form, requiring little transformation.
+
+Note that the schema files are located in two places:
+- spec/datasets/data_sources/definitions.yaml - reference types
+- spec/datasets/data_sources/data_sources_nodes.yaml - definition of each data_source
+      being loaded as a DataSource object
+- spec/collections/data_sources/data_sources_nodes.yaml - replication of the DataSource
+      object with the addition of a  _key property; all fields by reference to the
+      definitions in spec/datasets/data_sources
+
+TODO: A DRYer design would be for the schemas in spec/collections to use an allOf to
+extend the base DataSource type in spec/datasets.
+However, the schema for collection definition files located in spec/collections
+expects that the schemas in spec/collections be be an object.
```

Comment:
`be be` -- repeated word

Reply:
Gone now, but interestingly not an uncommon word duplication - can be found in the black and git libraries :)